packages/xo-server/src/api/server.mjs

```diff
@@ -40,6 +40,10 @@ add.params = {
     optional: true,
     type: 'string',
   },
+  fallbackAddresses: {
+    type: ['string', 'null'],
+    optional: true,
+  },
```

Comment:
I don't think it makes sense to expose for now, maybe we could simply save the addresses of all hosts after a successful first connection?